eth/contract/generate.go

```diff
@@ -0,0 +1,3 @@
+package contract
+
```

Comment:
would add a comment here about which version of the abi we use, maybe link the file in repo or specify the version of the contract.